### ISSUE TITLE
There is no product filtering in back-office by categories and brands.

### ISSUE THREAD

**Issue by RomanCharnashei** [simplcommerce/SimplCommerce#973]
There is no product filtering in back-office by categories and brands. This will definitely make it easier to manage the goods.
Is this so provided by design? Or you have not yet had enough time to implement it and you are open to contributing ?

**Comment by RomanCharnashei** [comment]
Hello again ), how about rework filter section like this:
![изображение](https://user-images.githubusercontent.com/23460571/108384714-0f85a680-721c-11eb-8912-fb9f978621bd.png)
This will allow us to add some extra filter options which are not included in the table .

**Comment by nqtuyen** [comment]
I think you should label is Craeted On, From, To. It seem have in localization table.

**Comment by RomanCharnashei** [comment]
> I think you should label is Craeted On, From, To. It seem have in localization table.
I am agree but this way one time binding turned out slightly tricky.
![изображение](https://user-images.githubusercontent.com/23460571/109167818-364d5b00-778f-11eb-801d-f761fca4a140.png)

**Comment by nqtuyen** [comment]
I find Culture & Resource class in Localization folder. I think it can help you translate Key and Value for your labels.

**Comment by RomanCharnashei** [comment]
> I find Culture & Resource class in Localization folder. I think it can help you translate Key and Value for your labels.

Yeah, I've learned how it works, too, but I don't understand how migration works, if I will add new records in sql script then how does it apply in already running applications  ?

**Comment by nqtuyen** [comment]
Ah, so you should add for running app's database.

Vào 18:44, Th 6, 26 thg 2, 2021 RomanCharnashei <notifications@github.com>
đã viết:

**Comment by RomanCharnashei** [comment]
so, does it mean it is done manually right ?

**Comment by nqtuyen** [comment]
I think you can add script to Static-xx.sql and run it to recreate
localization translate.

Vào 21:03, Th 6, 26 thg 2, 2021 RomanCharnashei <notifications@github.com>
đã viết:

**Comment by RomanCharnashei** [comment]
Any way, I am done with category and brand filtering and allocating into a separate filtering block. all changes in my fork.

**Comment by nqtuyen** [comment]
Great work.
if you want to separate filtering block, I think you should make with any critical that not present in header of table.
Thank

**Comment by RomanCharnashei** [comment]
> 
> 
> Great work.
> if you want to separate filtering block, I think you should make with any critical that not present in header of table.
> Thank

Yeah you are right, I have been added Brand and Category filtering
